mypy/semanal.py

```diff
@@ -4205,11 +4238,50 @@ def process_paramspec_declaration(self, s: AssignmentStmt) -> bool:
         if name is None:
             return False
 
-        # ParamSpec is different from a regular TypeVar:
-        # arguments are not semantically valid. But, allowed in runtime.
-        # So, we need to warn users about possible invalid usage.
-        if len(call.args) > 1:
-            self.fail("Only the first argument to ParamSpec has defined semantics", s)
+        n_values = call.arg_kinds[1:].count(ARG_POS)
+        if n_values != 0:
+            self.fail("Only the first positional argument to ParamSpec has defined semantics", s)
```

Comment:
It actually only takes one positional arg, the rest are keyword-only. So this error message seems imprecise; I think we can just say "Too many positional arguments to ParamSpec()"